app/models/category.rb

```diff
@@ -2,5 +2,5 @@ class Category < ActiveRecord::Base
   has_many :vendor_categories
   has_many :vendors, through: :vendor_categories
 
-  validates :category_name, presence: true
+  validates_presence_of :category_name
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.